### PR TITLE
SEO content/keywords updates from Revpartners

### PR DESCRIPTION
**Pull request by madebyguie**
Some copy/keyword updates done on few pages according with request from seo consultant

Added some keywords in the content, and updated title and meta description for each page

### CODE REVIEW

File: tyk-docs/assets/others/dashboard-swagger.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ info:
   title: Tyk Dashboard API
   version: 4.3.0
   description: |-
-    ## <a name="introduction"></a> Introduction
+    ## <a name="introduction"></a> Introduction to API Dashboards
```
Comment (letzya) [inline tyk-docs/assets/others/dashboard-swagger.yml]: this page is auto-gen, we need to remove it from the doc. it is from the  dashboard repo

File: tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 date: 2017-03-23T17:45:01Z
-title: Request Method Transform
+title: API Request Methods & API Gateway Transformation
+linktitle: Request Method Transform
+title: API Request Methods & API Gateway Transformation
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md:4]: 2 titles? 
Also why do we need linktitle?

File: tyk-docs/content/apim/open-source.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
 ---
-title: "Tyk Open Source"
+title: Open Source API Developer Portal | Tyk Documentation
```
Comment (letzya) [inline tyk-docs/content/apim/open-source.md:2]: dont understand this change. why dev portal?

File: tyk-docs/content/apim/open-source.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
+diffTitle: true
+diffTitleName: "Tyk Open Source API Management Solutions & Developer Portal"
+linktitle: Tyk Open Source
 date: 2020-06-24
 weight: 1
 tags: ["Tyk API Management", "Open Source", "API Gateway", "Tyk OSS"]
-description: "How to get started with the Tyk Open Source API Gateway"
+description: Tyk is an open source Enterprise API Gateway, supporting REST, GraphQL, TCP and gRPC protocols. Get started today.
```
Comment (letzya) [inline tyk-docs/content/apim/open-source.md:9]:
```suggestion
description: Tyk is an enterprise-grade open-source API Gateway, supporting REST, GraphQL, TCP and gRPC protocols. Get started today.
```

File: tyk-docs/content/apim/open-source.md [hunk 5]
```diff
@@ -1,9 +1,12 @@
 ---
-title: "Tyk Open Source"
+title: Open Source API Developer Portal | Tyk Documentation
+diffTitle: true
```
Comment (letzya) [inline tyk-docs/content/apim/open-source.md:3]: is this something new? googling didn't help

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 6]
```diff
@@ -1,8 +1,11 @@
 ---
 date: 2017-03-23T17:08:35Z
-title: Rate Limiting
+title: API Rate Limiting | How to Design API Rate Limiters
+diffTitle: true
+diffTitleName: API Rate Limiting
+linktitle: Rate Limiting
 tags: ["Rate Limiting", "Global limits", "Per API limits"]
-description: "How to use Global and per API rate limit throttling in Tyk"
+description: Tyk API will actively only allow a key to make x requests per y time period. This is critical to ensure your API does not get flooded with requests.
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md:8]: Can you explain this please? is this how we should start writing the descriptions? copy/paste a section from the page?

File: tyk-docs/content/planning-for-production/ensure-high-availability/service-discovery.md [hunk 7]
```diff
@@ -15,7 +18,7 @@ For example, you have ten Docker containers that are running the same service, a
 
 Now the Gateway would need to either be manually reconfigured, or, more appropriately, detect the failure and reconfigure itself.
 
-This is what the service discovery module does.
+This is what the API discovery service module does.
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/ensure-high-availability/service-discovery.md:21]: I think you be reverted. not need to add API there. What is the reason for this anyway?

File: tyk-docs/content/shared/tyk-gateway-features-include.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ Tyk Gateway is provided ‘Batteries-included’, with no feature lockout. Enabl
 
 Tyk Technologies uses the same API Gateway for all it’s applications. Protecting, securing, and processing APIs for thousands of organisations and businesses around the world. Ideal for Open Banking, building software in the clouds as well as exposing APIs to teams, partners & consumers.
 
-Built from the ground up to be the fastest API gateway on the planet. It does not depend on a legacy proxy underneath. It has no 3rd party dependencies aside from Redis for distributed rate-limiting and token storage. Tyk Gateway can also be deployed as part of a larger Full Lifecycle API Management platform Tyk Self-Managed which also includes Management Control Plane, Dashboard GUI and Developer Portal.
+Built from the ground up to be the fastest API gateway on the planet. It does not depend on a legacy proxy underneath. It has no 3rd party dependencies aside from Redis for distributed rate-limiting and token storage. Tyk Gateway can also be deployed as part of a larger Full Lifecycle API Management platform Tyk Self-Managed which also includes Management Control Plane, Dashboard GUI and Open Source Developer Portal.
```
Comment (letzya) [inline tyk-docs/content/shared/tyk-gateway-features-include.md]: why open source developer portal?

**Review by letzya — CHANGES_REQUESTED** [review]
We need to catchup

File: tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md [hunk 9]
```diff
@@ -1,6 +1,9 @@
 ---
 date: 2017-03-23T17:45:01Z
-title: Request Method Transform
+title: API Request Methods & API Gateway Transformation
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/request-method-transform.md]:
```suggestion
```

**Comment by dcs3spp** [comment]
Closing and re-opening for SEO project.

See ticket https://tyktech.atlassian.net/browse/DX-483

**Pull request closed without merging**